src/foam/graph/graph.js

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
Thanks for splitting. One last request; to be consistent, can you Capitalize this source file please.

Reply:
Looks like Git didn't let me do it the first time. Found a way around it